helpers/fetchApiData.ts

```diff
@@ -1,6 +1,7 @@
 export function validateListPath(path: string) {
+  // ()
   const regex =
-    /^lists\/[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}(\/(overview|activity|contributors)\?limit=\d+)?$/;
+    /^lists\/([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}|([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\/(contributors|stats\/(most-active-contributors|contributions-evolution-by-type))))\/??/;
```

Comment:
Updated this to accommodate new calls for lists. I created this originally because of the GitHub code analysis complaining about the list ID being passed in as data from the user, i.e. parsing it from the URL.